core-web/package.json

```diff
@@ -276,6 +276,9 @@
         "webpack": "^5.64.0",
         "webpack-bundle-analyzer": "^4.5.0"
     },
+    "resolutions": {
+        "pdfjs-dist": "^4.2.67"
+    },
```

Comment:
Just the update of `compodoc` didn't worked?

Reply:
No. Because due the lastest `@compodoc/compodoc@1.1.24` use `pdfjs-dist@2.12.313` the fix of pdfjs is `4.2.67`